app/services/edge_cache/bust_organization.rb

```diff
@@ -0,0 +1,17 @@
+module EdgeCache
+  class BustOrganization < Bust
+    def self.call(organization, slug)
+      return unless organization && slug
+
+      bust("/#{slug}")
+
+      begin
+        organization.articles.find_each do |article|
+          bust(article.path)
+        end
+      rescue StandardError => e
+        Rails.logger.error("Tag issue: #{e}")
```

Comment:
I'm not sure why we rescue errors here or call the errors 'Tag issue' 🤔

Reply:
🤔🤔🤔 :D

Reply:
That makes sense! Maybe @benhalpern has some insight as to where this came from originally.

My plan is to leave it for now. I'm trying to change as little as possible in the logic of the service so it's much easier to debug if something goes wrong when flipping the switch to using the service. By keeping the "copy/pasta" in place, the scope of debugging is much more narrow - I'm basically checking on the calling of the service. Then, once the services are up and running, we can always revisit for specific cleanups

Reply:
I merged this because it seems fine to maintain the code in the refactor, all else equal.

— But this particular rescue is the sort of thing that dates back to this _solo project rapidly do something to get it working_, so I have no specific idea why this is needed as such. Most likely can be removed if it seems like it can.